CONFIGURATION.md

```diff
@@ -84,8 +84,9 @@ The other placeholders are specified separately.
   # HTTP proxy server to use to connect to the targets.
   [ proxy_url: <string> ]
 
-  # The preferred IP protocol of the HTTP probe (ip4, ip6).
-  [ preferred_ip_protocol: <string> | default = "ip6" ]
+  # The IP protocol of the HTTP probe (ip4, ip6).
+  [ ip_protocol: <string> | default = "ip6" ]
```

Comment:
This will break any users currently setting this field, so I'm not sure about changing its name.

Reply:
Maybe support both, and print a warning about deprecating `preferred_ip_protocol` in the next release?

Reply:
I wanted to make it obvious that this is a behavior change.

But I like the idea of a deprecation warning.